Assets/Project/Scripts/Common/Components/UIs/MultiLanguageText.cs

```diff
@@ -45,14 +46,8 @@ protected override void Awake()
             base.Awake();
 
             // 言語変更するイベントを登録する
-            LanguageUtility.OnLanguageChange += OnLanguageChanged;
+            UserSettings.CurrentLanguage.Subscribe(_ => OnLanguageChanged()).AddTo(this);
```

Comment:
```suggestion
            UserSettings.CurrentLanguage.Subscribe(_ => {
                text = LanguageUtility.GetText(TextIndex);
            }).AddTo(this);
```

にして、 `private void OnLanguageChanged()` を消そう。あと、`ReactiveProperty` は明示的に指定しない限り、初期値が発行されるから、すぐ下の行の処理はいらないはず（ちょっと確認してみて）

Reply:
a561301
初期値の発行は困ることはないし、むしろ安全のためだと思うのでそのままでいいと思います

Reply:
だとしたら初期値を発行しないようにしよう。初期値を発行されるのに、安全のために冗長なコードを増やすのはあまり賛成じゃないです

Reply:
0c863be